update_action.py

```diff
@@ -0,0 +1,42 @@
+"""
+  Scans directories for files that satisfy a specific mask and updates
+  the actions' tags.
+  python3 update_action.py --input “one/*.y*ml two/*.y*ml three/*.y*ml” --name mlflow
+"""
+
+import argparse
+import glob
+import re
+from github import Github
+from pathlib import Path
+
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("-i", "--inputs", type=str, help="Input directories to scan")
+
+parser.add_argument("-n", "--name", type=str, help="Name of the action")
+
+
+# def glob_re(pattern, strings):
+#     return filter(re.compile(pattern).match, strings)
+
+args = parser.parse_args()
+inputs = args.inputs.split(" ")
+name = args.name
+g = Github()
+repo = g.get_repo(f"neuro-actions/{name}")
+tag = list(repo.get_releases())[-1].title
+# tag = '2.3.4'
+pattern = re.compile(
+    f"gh:neuro-actions/{name}" + r"@[a-zA-Z]{1}\d{1,2}\.\d{1,2}\.\d{1,3}"
```

Comment:
The scheme in action reference might be not only `gh:`, but also `github:`: [reference](https://neu-ro.gitbook.io/neuro-flow/reference/live-workflow-syntax#jobs-less-than-job-id-greater-than-action).
Let's support both schemes in the script.

Also, let's parameterize the owner name, `neuro-actions`, since some actions might be in the user's repo.

Reply:
@dmitrynikolayev , the same here. What do you think about the cases, when the action is hosted under a different repository?